### PR TITLE
Update discord.js

### PR DESCRIPTION
**Pull request by ndr3w221**

### CODE REVIEW

File: example/discord.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
-// Install discord.js before running this!
-const Discord = require('discord.js');
-const ytdl    = require('..');
+// Install discord.js, ffmpeg and ytdl-core before running this!
+const {Client} = require('discord.js');
```
Comment (fent) [inline example/discord.js]: Why import `Client` directly?

File: example/discord.js [hunk 2]
```diff
@@ -29,3 +28,6 @@ client.on('message', message => {
       });
   }
 });
+
+
+client.login(clientToken);
```
Comment (fent) [inline example/discord.js]: Why move `.login()` to end when it's asynchronous?

File: example/discord.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
-// Install discord.js before running this!
-const Discord = require('discord.js');
-const ytdl    = require('..');
+// Install discord.js, ffmpeg and ytdl-core before running this!
```
Comment (fent) [inline example/discord.js]: Might as well list `node-opus` too

File: example/discord.js (hunk 1 above)
Comment (ndr3w221) [reply example/discord.js]: why not :) also it makes it faster

File: example/discord.js (hunk 2 above)
Comment (ndr3w221) [reply example/discord.js]: its more human understandble

File: example/discord.js (hunk 3 above)
Comment (ndr3w221) [reply example/discord.js]: oh, i forgot about it :P

**Comment by ndr3w221** [comment]
here you go, node-opus

**Comment by fent** [comment]
Thanks!

**Pull request merged**